recipes/mlip/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "mlip" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/ashapeev/mlip-1-legacy/-/archive/master/mlip-1-legacy-master.tar.gz
+  sha256: b7b8624ea9e1ee57bdc083e214c607c0de8a87643dcdc8d56c6c7084a4109f68
+  patches:
+    - openblas.patch
```

Comment:
Has this patch been submitted upstream?

Reply:
The patch is mainly for using the central blas version from conda - therefore searching in `${PREFIX}/lib` - rather than using the openblas version which is provided with the Sourcecode.

Reply:
Right though it would still be useful to have them include some way to specify the BLAS to link against right? How about an issue at least?

Reply:
Sure I can definitely open an issue upstream.

Reply:
https://gitlab.com/ashapeev/mlip-1-legacy/-/issues/1